dashboard.js

```diff
@@ -0,0 +1,564 @@
+// Third-party dependencies
+const fs = require('fs')
+const Mustache = require('mustache')
+const Validator = require('express-validator')
+const session = require('express-session')
+const cookieParser = require('cookie-parser')
+
+// In-house dependencies
+const { helpers } = require('brave-alert-lib')
+const { ALERT_TYPE } = require('brave-alert-lib')
+const db = require('./db/db')
+
+const clientPageTemplate = fs.readFileSync(`${__dirname}/mustache-templates/clientPage.mst`, 'utf-8')
+const landingCSSPartial = fs.readFileSync(`${__dirname}/mustache-templates/landingCSSPartial.mst`, 'utf-8')
+const landingPageTemplate = fs.readFileSync(`${__dirname}/mustache-templates/landingPage.mst`, 'utf-8')
+const locationsCSSPartial = fs.readFileSync(`${__dirname}/mustache-templates/locationsCSSPartial.mst`, 'utf-8')
+const locationsDashboardTemplate = fs.readFileSync(`${__dirname}/mustache-templates/locationsDashboard.mst`, 'utf-8')
+const locationFormCSSPartial = fs.readFileSync(`${__dirname}/mustache-templates/locationFormCSSPartial.mst`, 'utf-8')
```

Comment:
unrelated to the changes in this PR, but what's the meaning of "Partial" in this variable/filename? Is it because this is the CSS we're adding *on top* of the stylesheet we inherit in the template page?

Reply:
The name "Partial" is what Mustache calls it (https://github.com/janl/mustache.js#partials). It's the term used for any file that doesn't product the full HTML. Right now it's been used for things that are used in multiple HTML pages, like the CSS and navigation bar.